shell/platform/darwin/common/framework/Source/FlutterStandardCodecHelper.h

```diff
@@ -62,6 +73,27 @@ CFTypeRef FlutterStandardCodecHelperReadValueOfType(
     CFTypeRef (*ReadTypedDataOfType)(FlutterStandardField, CFTypeRef),
     CFTypeRef user_data);
 
+//////////////////////////////////////////////
```

Comment:
What is this line for?

Reply:
Just a split between the reader functions and the writer functions.

Reply:
Maybe Something more descriptive? I'm not sure if there are any c++ header standard separator like #pragma in Objective C.

Reply:
I added comments that will group them together in our doxygen output.